src/ui/errorBoundary/ErrorBoundary.types.ts

```diff
@@ -0,0 +1,8 @@
+import {
```

Comment:
what's the point of reexporting those?

Reply:
The idea is that this way we don't relay on the `react-error-boundary` package as much, so there will be less trouble with expanding / modifying or writing our own implementation of `ErrorBoundary` component and we won't have to update imports in every place it's used